feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/PreviewScreen.kt

```diff
@@ -106,6 +108,85 @@ fun PreviewScreen(
                 }
             )
 
+            QuickSettingsUi(
```

Comment:
nit: maybe we should rename this to `QuickSettingsScreen` to remain consistent with existing convention?

Reply:
Done.